alt_e2eshark/utils/report.py

```diff
@@ -5,36 +5,131 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 import json
 import io
+from typing import List, Dict
 
-def save_dict(status_dict, status_dict_json):
+SUMMARY_STAGES = [
+    "Setup",
+    "IREE Compilation",
+    "Gold Inference",
+    "IREE Inference Invocation",
+    "Inference Comparison",
+]
+
+STAGE_SIMPLIFICATION = {
+    "setup": "Setup",
+    "import_model": "IREE Compilation",
+    "preprocessing": "IREE Compilation",
+    "compilation": "IREE Compilation",
+    "construct_inputs": "Gold Inference",
```

Comment:
Should we "Native ORT Inference" or "Reference ORT Inference", so it is clear what we are running here?

Reply:
Sure. I'm half split on this since the test suite was supposed to be generic enough to handle other frameworks. Let's see what Kumar thinks about the naming.